cmake/add_rpath_for_python.cmake

```diff
@@ -2,6 +2,7 @@
 # Make the SWIG generated Python interface modules more portable by using rpath for the the Python dynamic library.
 string(REPLACE "${LIST_SEPARATOR}" ";" SHARED_OBJECT_MODULES "${SHARED_OBJECT_MODULES}")
 foreach(_M ${SHARED_OBJECT_MODULES})
-  execute_process(COMMAND ${INSTALL_NAME_TOOL_EXE} -change "/Library/Frameworks/Python.framework/Versions/${Python_VERSION_MAJOR}.${Python_VERSION_MINOR}/Python" @rpath/${Python_VERSION_MAJOR}.${Python_VERSION_MINOR}/Python ${_M})
+  execute_process(COMMAND ${INSTALL_NAME_TOOL_EXE} -change "/Library/Frameworks/Python.framework/Versions/${Python_VERSION_MAJOR}.${Python_VERSION_MINOR}/Python" @rpath/Python.framework/Versions/${Python_VERSION_MAJOR}.${Python_VERSION_MINOR}/Python ${_M})
+  execute_process(COMMAND ${INSTALL_NAME_TOOL_EXE} -add_rpath /opt/homebrew/Frameworks ${_M})
```

Comment:
Why do we need this? Not to mention that we can't assume that people will have Homebrew installed on their Mac?

Reply:
We don't **need** it simply adds another path to search for the Python framework.  If you don't have a path then no issue.

Reply:
It is intended to increase portability.

Reply:
Yes, I eventually gathered that. All good then.